cost/aws/burstable_instance_cloudwatch_credit_utilization/README.md

```diff
@@ -0,0 +1,35 @@
+## AWS Burstable Instance CloudWatch Utilization Policy
+
+### What it does
+
+This Policy Template gathers AWS CloudWatch data for instances on 30 day intervals. Information on Burst Credits can be found [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-performance-instances-monitoring-cpu-credits.html). This policy will then take the appropriate actions based on which check fails and resize the instance. This is meant to be run as a monthly policy. 
```

Comment:
how is this a monthly policy.  there isn't a frequency for Monthly.  why does it need to run monthly.  couldn't it run as frequently as you want.  regardless of the frequency it should only the the values for the last 30 days.